tests/php/Domain/Services/CreateAccount.php

```diff
@@ -190,11 +194,23 @@ public function invalid_email_data() {
 	public function test_no_account_requested() {
```

Comment:
Should rename this to `test_no_account_created` to cover the new use case.

Reply:
Leaving this as is for now as the new test case isn't live yet (commented `test with explicitly turning off global registration`).

Reply:
Test is back! So we should rename :)